cli/src/highlight.rs

```diff
@@ -399,11 +401,27 @@ pub fn html(
     })?;
 
     let mut renderer = HtmlRenderer::new();
-    renderer.render(events, source, &move |highlight| {
-        theme.styles[highlight.0]
-            .css
-            .as_ref()
-            .map_or_else(|| "".as_bytes(), |css_style| css_style.as_bytes())
+    renderer.render(events, source, &move |highlight, output| {
+        if inline_styles {
+            output.extend("style='".as_bytes());
+            output.extend(
+                theme.styles[highlight.0]
+                    .css
+                    .as_ref()
+                    .map_or_else(|| "".as_bytes(), |css_style| css_style.as_bytes()),
+            );
+            output.extend("'".as_bytes())
```

Comment:
Consistency needed where applicable with the b"" syntax used in the else block below